### DISCUSSION TITLE
Save file

### DISCUSSION THREAD

**Discussion by Aizgo** [rocksdanister/lively#479] [Q&A]
How can i change the file i save the wallpapers?

**Comment by rocksdanister** [comment]
I don't understand, can you explain in more detail?

**Reply by Aizgo** [reply]
when i go to setting it doesn't let me change the file or the disc i save my wallpapers

**Reply by rocksdanister** [reply]
This is only supported for installer version of Lively, you cannot do this in store version.

Further read:
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions

**Reply by Aizgo** [reply]
How can I get the installer version?
